src/main/java/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -130,6 +130,8 @@ public void draw(final Rectangle bounds, final GameData data, final Graphics2D g
                 null);
           }
           break;
+        default:
+          throw new AssertionError("unknown unit flag draw mode: " + drawUnitNationMode);
```

Comment:
All current `UnitFlagDrawMode` enum values are handled, so the default case will only be triggered if a new enum value is added without adding a corresponding case.